### PR TITLE
Remove blood immunity from p.v model

### PR DESCRIPTION
**Pull request by RJSheppard**
p.f models blood immunity impact on new infections, while p.v does not. An if statement is added to the human_infection.R code to reflect this.

IB immunity is therefore also redundant, these variable assignments, immunity decay, boosts and rendering are removed when parasite == "vivax".

The blood immunity parameters have now been removed from the vivax parameter list, and documentation has been adjusted to reflect this.

### CODE REVIEW

File: R/processes.R [hunk 1]
```diff
@@ -169,6 +175,12 @@ create_processes <- function(
   # =========
   # Rendering
   # =========
+  
+  imm_var_names <- c('ica','icm','id','iva','ivm')
```
Comment (pwinskill) [inline R/processes.R:179]:
```suggestion
  imm_var_names <- c('ica', 'icm', 'id', 'iva', 'ivm')
```

File: R/processes.R [hunk 2]
```diff
@@ -169,6 +175,12 @@ create_processes <- function(
...
+  if(parameters$parasite == "falciparum"){
+    imm_var_names <- c(imm_var_names,'ib')
```
Comment (pwinskill) [inline R/processes.R:181]:
```suggestion
    imm_var_names <- c(imm_var_names, 'ib')
```

**Review by pwinskill — APPROVED** [review]
Two tiny formatting suggestions

**Review by giovannic — APPROVED** [review]

**Pull request merged**